advanced/Scripts/webpage.sh

```diff
@@ -401,22 +401,37 @@ SetWebUILayout() {
     change_setting "WEBUIBOXEDLAYOUT" "${args[2]}"
 }
 
+CheckUrl(){
+    local url validUrl regex
+    regex="^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$"
```

Comment:
Did we use this regex before or is it a new one?

Reply:
It's a stack-exchange special, of course. Have we ever validated URLs in bash before? I could not see anything.

Reply:
I don't know, could you add the SE URL there so we can find the source more easily when needed in the future?